helm-charts/templates/data-plane/gateway-components/gateway-runtime/gateway-runtime-deployment.yaml

```diff
@@ -168,7 +170,7 @@ spec:
             - name: ROUTER_CLUSTER
               value: "default_cluster"
             - name: ROUTER_LABEL
-              value: "Default"
+              value: "default"
```

Comment:
don't we need to parameterize all label envs to `.Values.wso2.apk.dp.gateway.name`

Reply:
resolved